### DISCUSSION TITLE
Catching error from requests and displaying to the user.

### DISCUSSION THREAD

**Discussion by priyamannur** [taver-ca/Taver_Hackathon#83] [Ideas]
Hello!  We could add up an error box displaying the appropriate error, when the user enters an invalid URL in spotify playlist. Right now it does not display anything, only in the developers tools one can make out.
So do let me know about this.

**Comment by sam9116** [comment]
I agree, some feedback is needed when user enters an invalid playlist url.
https://github.com/taver-ca/Taver_Hackathon/issues/85

**Comment by sam9116** [comment]
https://github.com/taver-ca/Taver_Hackathon/issues/85 is complete
